scripts/variantstore/wdl/extract/build_build_base_docker.sh

```diff
@@ -2,7 +2,7 @@
 
 if [ $# -lt 1 ]; then
     echo "USAGE: ./build_build_base_docker.sh [DOCKER_TAG_STRING] [OPTIONAL:LATEST]"
-    echo " e.g.: ./build_build_base_docker.sh \$(date -I)-alpine-build-base"
+    echo " e.g.: ./build_build_base_docker.sh \$(date -Idate)-alpine-build-base"
```

Comment:
Weird-looking but intentional. From the `date` man page on my machine:

```
     -I[FMT]
             Use ISO 8601 output format.  FMT may be omitted, in which case the default is date.  Valid FMT values are date, hours, minutes, and seconds.  The date and time is
             formatted to the specified precision.  When FMT is hours (or the more precise minutes or seconds), the ISO 8601 format includes the timezone.
```

I think what we saw with @RoriCremer last week was that she had a different version of `date` where the `FMT` was mandatory. It would be great to confirm that `date -Idate` does work on Rori's machine.

Reply:
hahaha yes good idea
<img width="582" alt="Screen Shot 2023-04-18 at 4 08 18 PM" src="https://user-images.githubusercontent.com/6863459/232893337-d64fd17a-2220-44c2-8b28-c635ecddfd8a.png">